rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -299,6 +302,96 @@ void gs_frame::flip(draw_context_t, bool /*skip_frame*/)
 	}
 }
 
+void gs_frame::take_screenshot(std::vector<u8> ss_data, const u32 ss_width, const u32 ss_height, bool vulkan_buf)
```

Comment:
Technically the vulkan_buf argument is just a format. Since we only have RGBA and BGRA, maybe just name it is_bgra since other APIs can also supply data in BGRA format.